src/main/java/io/vertx/ext/hawkular/examples/DatagramServerExample.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2015 Red Hat, Inc.
+ *
+ *  All rights reserved. This program and the accompanying materials
+ *  are made available under the terms of the Eclipse Public License v1.0
+ *  and Apache License v2.0 which accompanies this distribution.
+ *
+ *  The Eclipse Public License is available at
+ *  http://www.eclipse.org/legal/epl-v10.html
+ *
+ *  The Apache License v2.0 is available at
+ *  http://www.opensource.org/licenses/apache2.0.php
+ *
+ *  You may elect to redistribute this code under either of these licenses.
+ */
+package io.vertx.ext.hawkular.examples;
+
+import io.vertx.core.Vertx;
+import io.vertx.core.VertxOptions;
+import io.vertx.core.datagram.DatagramSocket;
+import io.vertx.ext.hawkular.VertxHawkularOptions;
+
+import java.util.Date;
+
+/**
+ * @author Thomas Segismont
+ */
+public class DatagramServerExample {
+
+  public static void main(String[] args) {
```

Comment:
I will do that when I'll write the new examples as part of my documentation work. I had written these examples in the beginning for manual testing purposes.